app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -158,6 +157,7 @@ class LuciBuildService {
     }
 
     final List<Request> requests = <Request>[];
+    final github.GitHub githubClient = await config.createGitHubClient(slug.owner, slug.name);
```

Comment:
Can we check if a token is expired?

Reply:
We can if we create a wrapper class for the github client but most likely not needed. The token is generated per request and in this particular case the problem is we were creating the client, sending some RPCs and after we got the responses from the RPCs we were using the token. With this change the token will be be used immediately after it was created.